test/e2e/common/host_path.go

```diff
@@ -111,6 +112,71 @@ var _ = framework.KubeDescribe("HostPath", func() {
 			"content of file \"" + filePathInReader + "\": mount-tester new file",
 		})
 	})
+
+	It("should support existing directory subPath [Volume]", func() {
+		volumePath := "/test-volume"
+		subPath := "sub-path"
```

Comment:
I just chagned the value in "should support existing file subPath", idk if that test is too confusing now